src/styles.scss

```
@@ -0,0 +1,131 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  margin: 0;
+  font-family: 'Nunito', sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)